src/main/java/uk/gov/companieshouse/pscdataapi/service/CompanyPscService.java

```diff
@@ -687,4 +687,11 @@ private boolean hasActivePscExemptions(String companyNumber) {
                                                 .anyMatch(e -> e.getExemptTo()==null)))).isPresent();
     }
 
+    private void deltaAtCheck(String requestDeltaAt, PscDocument document) {
+        if (isDeltaStale(requestDeltaAt, document.getDeltaAt())) {
+            logger.error("Stale delta received; request delta_at: [%s] is not after existing delta_at: [%s]".formatted(
+                    requestDeltaAt, document.getDeltaAt()), DataMapHolder.getLogMap());
+            throw new ResourceStateConflictException("Stale delta for delete");
```

Comment:
Should use a hand-made conflict exception, ideally.

Reply:
done